tests/func/test_remote.py

```diff
@@ -257,3 +257,22 @@ def test_modify_missing_remote(dvc):
 
     with pytest.raises(ConfigError, match=r"unable to find remote section"):
         remote_config.modify("myremote", "gdrive_client_id", "xxx")
+
+
+@pytest.mark.parametrize(
```

Comment:
Solution is for config in general, but tests are for `remote` specifically, not nice 🙂Also not clear why are you testing CLI again and not API?

Reply:
My bad, that's a left over. Those tests are not relevant since they were replaced for the ones in `test_config.py`

Reply:
:see_no_evil: I got confused with the other pull-request:

https://github.com/iterative/dvc/pull/3193/files#diff-dcfb923229ccfbba150bdd77494f46eeR96-R97

Reply:
I prefer testing the CLI in this case, it is more complete, but if you want, I use the API.

Reply:
@mroutis CLI is a dumb wrapper for API. We always test API, so let's do the same here. If you want to check that CLI returns proper codes on corresponding exceptions, just add a unit test to `tests/unit/command/`.